.circleci/config.yml

```diff
@@ -2076,21 +2076,13 @@ workflows:
           requires:
             - test
             - mock_e2e_tests
-            - graphql_e2e_tests
             - integration_test
-            - done_with_pkg_linux_e2e_tests
             - amplify_sudo_install_test
-            - amplify_console_integration_tests
-            - amplify_migration_tests_latest
-            - amplify_migration_tests_v4
-            - amplify_migration_tests_v4_30_0
             - github_prerelease_install_sanity_check
```

Comment:
You won't need the github release jobs either if you are deploying to npm

Reply:
Good point, removed.